### DISCUSSION TITLE
Help me install Tailwind on my VS code

### DISCUSSION THREAD

**Discussion by phuc1111x** [tailwindlabs/tailwindcss#2038] [Help]
I can not install tailwind on my VS code, does somebody know the reason?
![image](https://user-images.githubusercontent.com/67866003/87843118-fca0a180-c8db-11ea-9a67-fd2d642ef97a.png)

**Comment by estevanmaito** [comment]
Hey @phuc1111x welcome!

I see you're doing the Daily UI challenge, right?

There are some problems with your approach to use Tailwind, and if you're just starting with web dev, I would recommend that you start with the [CDN build (visit docs)](https://tailwindcss.com/docs/installation/#using-tailwind-via-cdn). It's the faster way to start and you don't need to run `npm` commands.

### What is wrong with your code

To install Tailwind, you would need `npm`, which comes with Node.js. You can find [more here](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).

With npm installed, then you could follow Tailwind's [installation instructions](https://tailwindcss.com/docs/installation/#1-install-tailwind-via-npm).

In any case (you're starting or experienced), the links above can point you to the right direction.

Cheers
